TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -55,6 +55,12 @@ public class Workspace
         public EventHandler<string> WarningTrigger { get; set; }
         public Queue<MessageActionWrapper> MessagesActionsQueue { get; private set; }
         private Func<string, Uri, bool> _Logger;
+        public List<string> CustomAnalyzerFiles { get; set; }
```

Comment:
The property is not used, you can remove it.

Reply:
done in 78ea44d